deploy/hooks.js

```diff
@@ -0,0 +1,118 @@
+const mime = require("mime-types");
+const baseDeploy = require("lifeomic-deploy");
+const walkDir = baseDeploy.util.fs.walkDir;
+const formatBytes = require("bytes");
+const pAll = require("p-all");
+const fs = require("fs").promises;
+
+exports.afterDeployTerraform = async (env) => {
+  const S3_BUCKET = `${env["name"]}-security-policy-templates`;
+  const AWS = baseDeploy.getAws().getAwsSdk();
+  const s3 = new AWS.S3();
+  const config = baseDeploy.getConfig();
+  const templatesDir = config.getProjectDir() + "/templates/";
+  const assets = [];
+
+  await walkDir(templatesDir, {
+    async onFile(file, stat) {
+      // Sanity check before blindly removing the prefix by length below
+      if (!file.startsWith(templatesDir)) {
+        throw new Error(
+          `The file ${file} did not start with the expected path ${templatesDir}`
+        );
+      }
+      assets.push({
+        fullPath: file,
+        relativePath: file.substring(templatesDir.length),
+        contentType: mime.lookup(file) || "text/plain",
+      });
+    },
+  });
+
+  console.log("Number of static assets: " + assets.length);
+  console.log("Uploading static assets to S3...");
+
+  const uploadAsset = async (asset) => {
+    const { body, encoding } = asset;
+    const key = asset.relativePath;
+    console.log("bucketname", S3_BUCKET);
+    try {
+      await s3
+        .upload({
+          Bucket: S3_BUCKET,
```

Comment:
FYI, if you don't specify encryption when uploading then the default encryption of the bucket will be used. Since you didn't specify default encryption, these S3 bucket objects will be unencrypted.